.github/workflows/delete_backport_branch.yml

```diff
@@ -0,0 +1,15 @@
+name: Delete merged branch of the backport PRs
+on: 
+  pull_request:
+    types:
+      - closed
+  
+jobs:
+  delete-branch:
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.pull_request.head.ref,'backport/')
+    steps:
+      - name: Delete merged branch
+        uses: SvanBoxel/delete-merged-branch@main
```

Comment:
The action `SvanBoxel/delete-merged-branc` is pointed to `@main`, if a bad change gets deployed tomorrow we will automatically run it, we should change this reference to a tag or commit id to be sure we are only running the changes we want.

Reply:
Makes sense! Updated with commit id.